test/gtest/uct/test_p2p_am.cc

```diff
@@ -489,4 +493,4 @@ UCS_TEST_P(uct_p2p_am_misc, am_max_short_multi) {
     EXPECT_EQ(UCS_OK, status);
 }
 
-UCT_INSTANTIATE_TEST_CASE(uct_p2p_am_misc)
+UCT_INSTANTIATE_NO_SELF_TEST_CASE(uct_p2p_am_misc)
```

Comment:
then lets add timeout to "am_max_short_multi" that if after ~1sec the resources are not exhausted, the test passes.
this way it could handle any transport which always has resources